app/scripts/modules/kubernetes/src/serverGroup/serverGroupTransformer.service.spec.ts

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+import { KubernetesV2ServerGroupTransformer } from './serverGroupTransformer.service';
+import { Application } from '@spinnaker/core';
+import { IKubernetesServerGroup } from '../interfaces';
+
+describe('KubernetesV2ServerGroupTransformer', function() {
+  it('normalized server group name', async () => {
+    const transformer = new KubernetesV2ServerGroupTransformer();
+    const ACCOUNT = 'myaccount';
+    const LOCATION = 'myns';
+    const SG_MANAGER = 'sgmanager';
+    const KIND = 'daemonSet';
+
+    const sgMgrRef = { account: ACCOUNT, location: LOCATION, name: SG_MANAGER };
+    const sgAccount: IKubernetesServerGroup = ({
```

Comment:
Can drop the left-hand side `IKubernetesServerGroup` typing unless you think it's more readable that way.